### DISCUSSION TITLE
Vercel: Can't deploy anymore after update to tRPC 10

### DISCUSSION THREAD

**Discussion by Michael-Reich** [trpc/trpc#4652] [Q&A]
I get this error after updating my whole application to the last version of tRPC.

Error: The Serverless Function "api/trpc/[trpc]" is 56.71mb which exceeds the maximum size limit of 50mb. Learn More: https://vercel.link/serverless-function-size

You guys know how I could lower that mb? Because I don't now where this comes from.

**Comment by KATT** [comment]
This come from your own code or other deps.

You can split it up by doing something like this: https://github.com/trpc/trpc/tree/main/examples/soa

**Comment by Michael-Reich** [comment]
Thank you for your response.

The strange thing is, I removed one import pacakge from a route and the size did not change at all. Do you know by chance what affects the size the most? The amount of routes?

**Reply by KATT** [reply]
Likely big routers that import things. There should be very little diff between v9 and v10 so I am very surprised you hit this as you migrated
